plugins/MauticCrmBundle/Integration/PipedriveIntegration.php

```diff
@@ -18,14 +18,20 @@
 use Monolog\Logger;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
 use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Component\HttpFoundation\Session\Session;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 use Symfony\Component\Routing\Router;
 use Symfony\Component\Translation\TranslatorInterface;
 
+/**
+ * @deprecated will be removed in Mautic 5. See DEPRECATION_MESSAGE bellow.
```

Comment:
```suggestion
 * @deprecated will be removed in Mautic 5. See DEPRECATION_MESSAGE below.
```

Reply:
Fixed in https://github.com/mautic/mautic/pull/12364/commits/1a57bda2081db45d456878938f7a027137745b7d